python/graphstorm/dataloading/dataset.py

```diff
@@ -553,6 +554,13 @@ def __init__(self, graph_name, part_config, train_etypes, eval_etypes=None,
                                                  decoder_edge_feat,
                                                  lm_feat_ntypes=lm_feat_ntypes,
                                                  lm_feat_etypes=lm_feat_etypes)
+        if self._train_etypes == [DEFAULT_ETYPE]:
+            assert self._g.ntypes == [DEFAULT_NTYPE] and \
+                   self._g.etypes == [DEFAULT_ETYPE[1]], \
```

Comment:
Should not DGL etypes be canonical etypes?

Reply:
Nope. For DGL graph, the input is just [DEFAULT_ETYPE[1]]

Reply:
Can you add a comment here?